src/integrator/cuda/cuda_eval_denvars.cu

```diff
@@ -187,9 +187,9 @@ void eval_uvars_gga_device( size_t           ntasks,
                             XCTaskDevice<T>* tasks_device,
                             cudaStream_t     stream ) {
 
-  dim3 threads(warp_size, max_warps_per_thread_block, 1);
-  dim3 blocks( util::div_ceil( max_nbf , threads.x ),
-               util::div_ceil( max_npts , threads.y ),
+  dim3 threads( warp_size, max_warps_per_thread_block / 2, 1 );
```

Comment:
Is there a reason this won't run on 1024? Is it a register issue?

Reply:
It should run with a 32x32 thread-block, but I found 32x16 to be a little faster